File: src/main/java/no/nav/melosys/eessi/models/sed/nav/Vedtak.java

package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;
import lombok.Data;


@SuppressWarnings("unused")
@JsonInclude(Include.NON_NULL)
@Data
public class Vedtak {

    private String artikkelforordning;

    private String datoforrigevedtak;

    private String erendringsvedtak;

    private Periode gjelderperiode;

    private String gjeldervarighetyrkesaktivitet;

    private String land;

    private String eropprinneligvedtak;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Vedtak.kt
package no.nav.melosys.eessi.models.sed.nav

abstract class Vedtak {
    var datoforrigevedtak: String? = null
    var eropprinneligvedtak: String? = null // RINA regler: Kan bare sette "ja" eller null (default: null, som betyr nei)
    var erendringsvedtak: String? = null // RINA regler: Kan bare sette "nei" eller null (default: null, som betyr ja)
}
